store/keyboard/index.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  createContext,
+  FC,
+  KeyboardEvent,
+  ReactElement,
+  useContext,
+  useState,
+} from "react";
+import { ALT_KEY } from "@/constants/AppConstants";
+import { KEYBOARD_CONTEXT } from "@/types/member_store";
+
+const KeyboardContext = createContext<KEYBOARD_CONTEXT>({isOptionKeyPressed : false});
+
+function isOptionKey(e: KeyboardEvent) {
+  return e.key === ALT_KEY;
+}
+
+export const KeyboardProvider: FC<{ children: ReactElement }> = ({
+  children,
+}): ReactElement => {
+  const [isOptionKeyPressed, setIsOptionKeyPressed] = useState(false);
+
+  const initialValue = {
+    isOptionKeyPressed,
+  };
+
+  return (
+    <KeyboardContext.Provider value={initialValue}>
+      <div
+        role="button"
```

Comment:
the role is a button, I see we are not adding the following events to handle:

1. onblur
2. onfocus
3. on enter